modules/govuk/manifests/apps/email_alert_service/rabbitmq_test_permissions.pp

```diff
@@ -28,25 +28,33 @@
   $amqp_user  = 'email_alert_service_test',
   $amqp_pass  = 'email_alert_service_test',
   $amqp_exchange = 'email_alert_service_published_documents_test_exchange',
-  $amqp_queue = 'email_alert_service_published_documents_test_queue',
+  $amqp_major_change_queue = 'email_alert_service_published_documents_test_queue',
+  $amqp_unpublishing_queue = 'email_unpublishing_test_queue',
```

Comment:
It might be good to be consistent with the naming, so `email_alert_service_unpublishing_test_queue`?

Reply:
done